nautobot/dcim/views.py

```diff
@@ -273,6 +276,105 @@ class LocationTypeUIViewSet(NautobotUIViewSet):
 #
 
 
+class RackGroupsFieldsPanel(object_detail.ObjectFieldsPanel):
```

Comment:
1) Perhaps `LocationRackGroupsPanel`?
2) `ObjectFieldsPanel` seems like the wrong base class here. Should it just be a `Panel` subclass instead?